web/app/css/pdf.css

```diff
@@ -131,3 +131,8 @@ figure.image figcaption {
     border: 1px solid gray;
     text-align: center;
 }
+
+/* This helps alignment for inline equations */
+img {
```

Comment:
this is too broad a selector

Reply:
I don't know a better solution.
Mpdf extracts the embedded `<svg>` to a temporary file xyz.svg and replaces the `<svg>` element with `<img src='xyz.svg'>`. There is no class or style transferred to `<img>`. [See the mpdf source](https://github.com/mpdf/mpdf/blob/3b21745a7b0178c14bf512d79b790a4351c16bce/src/Mpdf.php#L26936).

I tried to wrap it in a `<span class="mathjax"><svg>...</span>` (`.mathjax img {vertical-align: middle;}`) but it [seems like it is not implemented](https://mpdf.github.io/css-stylesheets/introduction.html):
> CSS attributes, used in stylesheets or in-line, can define:
> - most tags/elements e.g. `div`, `p`, `body`, `table`, `span`
> - class-names e.g. `p.mystylename { font-size:9pt; }`
> - id e.g. `div#style { font-size:9pt; }`
>
> ...
> 
> There is some support for ‘cascaded’ CSS e.g. `div.topic table.type1 td {...}`
> - `table`, `tr`, `th` and `td` will only be recognised as the last items (as above)
> - only ‘block’ elements (not ‘in-line’) can be included i.e. **`div.style1 a {...}` will not work, nor will `a#class1 {...}`**

I also realized that temporary files accumulate in `cache/mpdf/mpdf`. Need to monitor if the will be deleted or not.

Reply:
You can use an [attribute selector](https://www.w3.org/TR/selectors-3/#attribute-substrings) to match the src that begins with `/_tempSVG`:

~~~css
img[src^='/_tempSVG'] {}
~~~

Reply:
As for the cache it should be fine to leave it here and act its role of cache I guess.

Reply:
> ```css
> img[src^='/_tempSVG'] {}
> ```
and `img[src*='tempSVG'] {}` and `img[src$='svg'] {}`  don't work.
Attribute selectors are most likely not implemented.

Reply:
see: https://github.com/mpdf/mpdf/issues/134#issuecomment-220166048

Reply:
See https://github.com/mpdf/mpdf/issues/1404

Reply:
Opening an issue upstream is the best approach ;)

Reply:
Just add some tests to your PR on mpdf and it should be good ;)

Reply:
css img selector is still in but can be easily removed after PR https://github.com/mpdf/mpdf/pull/1405 is accepted and mpdf is bumped to next version